Lib/_pyio.py

```diff
@@ -1794,6 +1794,20 @@ def isatty(self):
         self._checkClosed()
         return os.isatty(self._fd)
 
+    def _isatty_openonly(self):
+        """Checks whether the file is a TTY using an open-only optimization.
+
+        Normally isatty always makes a system call. In the case of open() there
+        is a _inside the same python call_ stat result which we can use to
```

Comment:
what is "call_ stat"?

Reply:
My mental was Markdown italics for emphasis, but can remove. If want to keep, I should probably be doing reST (https://devguide.python.org/documentation/markup/#inline-markup, https://peps.python.org/pep-0287/).

Reply:
Usually, we don't use reST formatting in docstrings.